backend/lib/spree/backend/tabs/product/images.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spree/backend/tabs/product/base'
+
+module Spree
+  module Backend
+    module Tabs
+      class Product
+        class Images < Base
+          def name
+            'Images'
+          end
+
+          def presentation
+            view_context.t('spree.images')
+          end
+
+          def url
+            view_context.spree.admin_product_images_url(product)
```

Comment:
Could we pass the `spree` url proxy into the base class instead of using the `view_context`? Or try to make use of `Spree::Core::Engine.routes.url_helpers`

Reply:
Maybe add a `url_helpers` method on the base class

```rb
def url_helpers
  Spree::Core::Engine.routes.url_helpers
end
```

and make use of it here?

```rb
def url
  url_helpers.admin_product_images_path(product)
end
```